chapter_linear-networks/linear-regression-gluon_vn.md

```diff
@@ -268,29 +268,34 @@ Once we start working with much more complex models, Gluon's advantages will gro
 However, once we have all the basic pieces in place, the training loop itself is strikingly similar to what we did when implementing everything from scratch.
 -->
 
-*dịch đoạn phía trên*
+Bạn có thể thấy rằng việc thể hiện mô hình của chúng ta thông qua Gluon đòi hỏi một vài dòng lệnh tương đối.
+Chúng ta cần phải phân bổ các tham số một cách riêng lẻ, xác định hàm số mất mát của chúng ta , hay hạ gradient ngẫu nhiên.
+Một khi chúng ta làm với nhiều mô hình rất phức tạp , những lợi ích của Gluon sẽ tăng lên nhiều.
+Tuy nhiên, một khi chúng ta có tất cả các mảnh cơ bản ở đúng chỗ, vòng huấn luyện của chính bản thân rất giống với những gì chúng ta đã thực hiện tự lúc đầu.
```

Comment:
```suggestion
Tuy nhiên, một khi chúng ta có tất cả các mảnh cơ bản đặt đúng chỗ, vòng huấn luyện sẽ tương tự như những gì ta đã làm khi triển khai mọi thứ từ đầu.
```

Reply:
@honghanhh  @NhatQuanBui  "once we have all the basic pieces in place," mình nghĩ nên dịch thành "một khi ta nắm được rõ các điều cơ bản," nghe có vẻ ổn hơn ạ.

Reply:
```suggestion
Tuy nhiên, một khi ta nắm được rõ các điều cơ bản, vòng huấn luyện tự thân nó sẽ trông rất giống với những gì chúng ta tự thực hiện từ đầu.
```

Reply:
```suggestion
Khi chúng ta có đủ các mảnh ghép quan trọng, vòng huấn luyện sẽ rất giống với vòng huấn luyện chúng ta tự xây dựng mà không dùng Gluon.
```

Reply:
Notes: đừng đùng chữ `triển khai` để dịch chữ `implement`. Dùng chữ `xây dựng` á.